client/src/components/Header/Header.scss

```diff
@@ -15,13 +15,18 @@
   z-index: 1250 !important;
 }
 
-.header-font {
-  font-family: wtt-thin, Arial, sans-serif, "PT Sans";
-  color: white;
-  white-space: nowrap;
-  letter-spacing: -3px;
-  font-size: 1.8rem;
-  word-spacing: 0.6rem;
+// .header-font {
+//   font-family: wtt-thin, Arial, sans-serif, "PT Sans";
+//   color: white;
+//   white-space: nowrap;
+//   letter-spacing: -3px;
+//   font-size: 1.8rem;
+//   word-spacing: 0.6rem;
+// }
```

Comment:
Same w/ this commented code

Reply:
> @Tijani-zainab did you see how the logo behaves when you change the screen size to mobile? Even a higher resolution screen?
> 
> This looks great. Will look really good when the sidebar is implemented.

Hi @ri0nardo , the logo seems too large on mobile but I'll like to know if there's any design for it looks on mobile before I go ahead and resize. 

so, I'd get the perfect pixel.